src/components/AnalysisModal.js

```diff
@@ -137,6 +150,8 @@ export const AnalysisModal = Utils.withDisplayName('AnalysisModal')(
         await refreshApps(true)
       })
     })
+    const renderGalaxyModal = () => renderAppModal(GalaxyModalBase, tools.galaxy.label)
+    const renderCromwellModal = () => renderAppModal(CromwellModalBase, tools.cromwell.label)
```

Comment:
Wondering why we need these functions. Can't we inline the body where they are called?

Reply:
Totally don't need them… it just happened because of refactoring (originally there were 2 separate methods). Pushing up change.